tests/spec/w3c/headers-spec.js

```diff
@@ -250,19 +250,89 @@ describe("W3C — Headers", function() {
       const doc = await makeRSDoc(ops);
       expect($("#subtitle", doc).length).toEqual(0);
     });
-    
+  
     it("uses existing h2#subtitle as subtitle", async () => {
       const ops = makeStandardOps();
       ops.body = "<h2 id='subtitle'>This is a pre existing subtitle <code>pass</code></h2>" + makeDefaultBody();
+    
+      const doc = await makeRSDoc(ops);
       
+      const subTitleElement = doc.querySelectorAll("h2#subtitle");
+      expect(subTitleElement.length).toEqual(1);
+    
+      const confSubTitle = doc.defaultView.respecConfig.subtitle;
+    
+      if (confSubTitle) {
+        expect(confSubTitle).toEqual("This is a pre existing subtitle pass");
+      } else {
+        expect(confSubTitle).toEqual("");
+      }
+    
+      const [forInnerText] = subTitleElement;
+      expect(forInnerText.textContent).toEqual("This is a pre existing subtitle pass");
+    
+      const childElement = forInnerText.querySelector("code");
+      expect(childElement.textContent).toEqual("pass");
+    });
+  
+    it("overwrites conf.subtitle if it exists", async () => {
+      const ops = makeStandardOps();
+    
+      ops.body = "<h2 id='subtitle'>This is a pre existing subtitle <code>pass</code></h2>" + makeDefaultBody();
+    
+      const newProps = {
+        subtitle: "sub",
+      };
+    
+      Object.assign(ops.config, newProps);
+    
       const doc = await makeRSDoc(ops);
-
+    
+      const initialSubtitle = doc.defaultView.respecConfig.subtitle;
```

Comment:
nit: calling this "initialSubtitle" makes it sound like it could change. Maybe just:

```JS
 const { subtitle } = doc.defaultView.respecConfig;
```

Reply:
If you want the actual "initial subtitle", that's "ops.config.subtitle", as that's the one you passed. 

So, somewhere:

```JS
expect(doc.defaultView.respecConfig.subtitle).toEqual(ops.config.subtitle)
```

Reply:
I guess we don't need it now. But thanks for the information :sweat_smile: